### ISSUE TITLE
🐛 [Bug] Encountered bug "Unable to freeze tensor of type Int64/Float64 into constant layer" when using TRTorch

### ISSUE THREAD

**Issue by JuncFang-git** [NVIDIA/Torch-TensorRT#584]
##  Bug Description
I got a bug when i convert an operation :
heatmap = F.softmax(heatmap / self.temperature, dim=2)

Where heatmap is a tensor and self.temperature is 0.1.

It's works when I change it  to:
heatmap = F.softmax(heatmap / torch.tensor(self.temperature), dim=2)

The bug info:
terminate called after throwing an instance of 'trtorch::Error'
  what():  [Error thrown at core/conversion/converters/converter_util.cpp:160] Unable to freeze tensor of type Int64/Float64 into constant layer, try to compile model with truncate_long_and_double enabled

## To Reproduce

1.a= F.softmax(tensor / 0.1, dim=2)
1. Is ok for : a= F.softmax(tensor / torch.tensor(0.1), dim=2)
1. Also multiply have same bug : a= F.softmax(tensor * 0.1, dim=2)

## Expected behavior

Fix this bug and merge to master branch. Or make sure it is no problem.

## Environment

> I build the Trtorch recently(21.8.19) use the master branch.

 - TRTorch Version (8.0.1.6): 
 - PyTorch Version (libtorch 1.9.0):
 - OS (Ubuntu):
 - How you installed PyTorch (`libtorch):
 - Build command you used (bazel build //:libtrtorch --compilation_mode opt):
 - Are you using local sources or building from archives:
 - Python version: 3.8
 - CUDA version: 11.1
 - GPU models and configuration: GTX3070

## Additional context
I compile the model use below code:
`std::cout << "Load ts detector model ..." << std::endl;
    torch::jit::Module ts_detector_model;
    try{
        ts_detector_model = torch::jit::load(TS_DETECTOR_PATH);
    }
    catch (const c10::Error& e){
        std::cerr << "Error loading the model \n";
        return -1;
    }

    // convert trt detector
    std::cout << "Convert trt detector model ..." << std::endl;
    ts_detector_model.to(at::kCUDA);
    ts_detector_model.eval();

    std::vector<trtorch::CompileSpec::Input> inputs_d = {
      trtorch::CompileSpec::Input(std::vector<int64_t>({1, 3, 256, 256}), trtorch::CompileSpec::DataType::kHalf)};
    auto info_d = trtorch::CompileSpec(inputs_d);
    info_d.enabled_precisions.insert(trtorch::CompileSpec::DataType::kHalf);

    auto trt_detector_model = trtorch::CompileGraph(ts_detector_model, info_d);`

**Comment by narendasan** [comment]
Can you try adding? 

```c++
info_d.truncate_long_and_double = true;
```
